samples/bluetooth/st_ble_sensor/README.rst

```diff
@@ -0,0 +1,59 @@
+.. _bluetooth-st_ble_sensor:
+
+Bluetooth: ST BLE Sensor Demo
+#############################
+
+Overview
+********
+
+This application demonstrates BLE peripheral by exposing vendor-specific
+GATT services. Currently only button notification and LED service are
+implemented. Other BLE sensor services can easily be added.
+See `BlueST protocol`_ document for details of how to add a new service.
+
+Instructions for `ST Nucleo WB55RG`_ board
+
+This sample requires having the stm32wb5x_BLE_Stack_fw.bin binary flashed
+on the Wireless Coprocessor (Cortex M0+ core).
+The file stm32wb5x_BLE_Stack_fw.bin can be found on `STM32WB github`_
+
+Requirements
+************
+
+Instructions for `ST Nucleo WB55RG`_ board
+
+* `ST BLE Sensor Android app <ST BLE Sensor app>`_
+* A board with STM32WB55 like ST Nucleo WB55RG
+* You need to use `STM32CubeProgrammer`_ to flash the appropriate binary on M0+ core.
+* :ref:`Zephyr kernel environment configured for ST Nucleo WB55RG <nucleo_wb55rg_board>`
+
+Building and Running
+********************
+
+This sample can be found under :zephyr_file:`samples/bluetooth/st_ble_sensor` in the
+Zephyr tree.
+
+Instructions for `ST Nucleo WB55RG`_ board:
```

Comment:
No need to add link to extrernal doc for this board (or this should be done in board doc)

Reply:
done